applications/ecallmgr/src/ecallmgr_registrar.erl

```diff
@@ -381,6 +382,12 @@ init([]) ->
 handle_call('registrar_age', _, #state{started=Started}=State) ->
     wh_util:put_callid(?LOG_SYSTEM_ID),
     {'reply', wh_util:current_tstamp() - Started, State};
+handle_call({'import_registrations', Filename}, _, State) ->
+    wh_util:put_callid(?LOG_SYSTEM_ID),
+    {'reply', import_registrations_from_file(Filename), State};
```

Comment:
How about moving the file IO to the ```ecallmgr_maintenace.erl``` which would run in the sup process and not block the ecallmgr_registrar process? I will just pass the list in-memory between the ecallmgr proc and the sup proc. This will also allow to add support for stdin/out export and import from shell.